### ISSUE TITLE
Badge text not updates on change

### ISSUE THREAD

**Issue by efec89** [gorhom/react-native-animated-tabbar#39]
# Bug

I'm using mobx for states. I crated cart system. When cart count changes, badge not updating.

Cart.tsx file like this:

```
const CartSVG = ({ color, size }: SVGProps) => {
  const user = CounterStore.user;
  return (
      <View>
        <View style={styles.badge}>
          <Text style={styles.badgeText}>{user.basket}</Text>
        </View>
        <SvgXml xml={xml} width="25" height="24" />
      </View>
  );
};

export default CartSVG;
```

**Comment by gorhom** [comment]
Hi @efec89 , have you tried to wrapping it with ‘observer’ from mobx ?

**Comment by efec89** [comment]
My mobx side like this: 

```
import {observable, action, configure } from 'mobx';

configure({
  enforceActions:"observed"
})


class CartState{
    @observable user = {
        firstName: "John",
        lastName: "Doe",
        basket: 4
    }

    @action changeName(firstName, lastName) {
        this.user.firstName = firstName;
        this.user.lastName = lastName;
    }

    @action addBasket(){
        this.user.basket++;
    }

    @action removeBasket(){
      this.user.basket--;
    }
}
const store = new CartState();
export default store;

```

**Comment by efec89** [comment]
@gorhom  How can I use this icon file like class?

```
@observer
export default class CartSVG extends React.Component<Props> {

  constructor(props) {
    super(props);
  }
  render(){
    return(
        <View>
          <View style={styles.badge}>
            <Text style={styles.badgeText}>{user.basket}</Text>
          </View>
          <SvgXml xml={xml} width="25" height="24" />
        </View>
    )
  }
```

**Comment by gorhom** [comment]
i think you can do this in `mobx` 
```ts
import {observer} from 'mobx-react';

const CartSVG = observer(({ color, size }: SVGProps) => {
  const user = CounterStore.user;
  return (
      <View>
        <View style={styles.badge}>
          <Text style={styles.badgeText}>{user.basket}</Text>
        </View>
        <SvgXml xml={xml} width="25" height="24" />
      </View>
  );
});

export default CartSVG;
```

**Comment by efec89** [comment]
![Screenshot_1590242331](https://user-images.githubusercontent.com/6522288/82732590-c77e3580-9d16-11ea-8ab0-ffa62bbf7062.png)

I tried your code, this error shows.

**Comment by gorhom** [comment]
this been released with v1.5.1 🎉
